scripts/source-maintenance.sh

```diff
@@ -216,7 +238,25 @@ git grep "ifn?def .*_SQUID_" |
 #
 # Scan for file-specific actions
 #
-for FILENAME in `git ls-files`; do
+FORKPOINT=""
+FILES=""
+if test "x$ONLYCHANGEDSINCE" = "x" ; then
+    FILES=`git ls-files`
+fi
+if test "x$FILES" = "x" && test "x$ONLYCHANGEDSINCE" = "xfork" ; then
+    FORKPOINT=`git merge-base --fork-point upstream/master`
+    if test "x$FORKPOINT" = "x" ; then
+        echo "Could not identify fork point - sometimes it happens"
+        echo "Please specify commit-id explicitly"
+        exit 1
+    fi
+    ONLYCHANGEDSINCE="${FORKPOINT}"
+fi
+if test "x$FILES" = "x" && test "x$ONLYCHANGEDSINCE" != "x" ; then
+    FILES=`git diff "$ONLYCHANGEDSINCE" --name-only`
```

Comment:
Please use the git-documented parameter order and avoid quotes. It will allow for a lot more flexibility with the new `--only-changed-since` option:

```suggestion
    FilesToFormat=`git diff --name-only $BasePoint`
```

Reply:
Done in c17e37d6451e076095c90020a5156d1dbd6b0d8d